inlong-sdk/dataproxy-sdk-twins/dataproxy-sdk-python/demo/send_demo.py

```diff
@@ -0,0 +1,72 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+import sys
+
+import inlong_dataproxy
+
+# user defined callback function
+def callback_func(inlong_group_id, inlong_stream_id, msg, msg_len, report_time, ip):
+    print("******this is call back, print info******")
+    print("inlong_group_id:", inlong_group_id, ", inlong_stream_id:", inlong_stream_id)
+    print("msg_len:", msg_len, ", msg content:", msg)
+    print("report_time:", report_time, ", client ip:", ip)
+    print("******call back end******")
+    return 0
+
+def main():
+    if len(sys.argv) < 2:
+        print("USAGE: python send_demo.py config_example.json [inlong_group_id] [inlong_stream_id]")
+        return
+
+    inlong_api = inlong_dataproxy.InLongApi()
+
+    # step1. init api
+    if inlong_api.init_api(sys.argv[1]):
```

Comment:
It is recommended to print out the return code information to facilitate troubleshooting.

Reply:
Added printing of return codes in case of interface call errors.